src/gui/sdlmain.cpp

```diff
@@ -2876,6 +2881,11 @@ void Config_Add_SDL() {
 	Pint = Pmulti->GetSection()->Add_int("ysens",Property::Changeable::Always,100);
 	Pint->SetMinMax(-1000,1000);
 
+	pbool = sdl_sec->Add_bool("raw_mouse_input", on_start, false);
+	pbool->Set_help("Enable this setting to bypass mouse sensitivity and\n"
```

Comment:
@kcgen I am not sure about using the word "bypass" in here - perhaps "disable" would work better? How does this sentence read to a native English speaker?

Reply:
Bypass sounds good to me. It's the OS that's being bypassed, so I can only suggestion something like: *"Enable this setting to bypass your operating system's mouse sensitivity and acceleration settings."*

Reply:
I think this description is fine.

Reply:
Not sure where my prior comment went! 
Bypass is good; but suggest a small order change to the active voice: *"Enable this setting to bypass your operating system's mouse acceleration and sensitivity settings."*